api/server/backend/build/tag.go

```diff
@@ -37,7 +37,7 @@ func (bt *Tagger) TagImages(imageID image.ID) error {
 	for _, rt := range bt.repoAndTags {
 		// TODO @jhowardmsft LCOW support. Will need revisiting.
 		platform := runtime.GOOS
-		if platform == "windows" && system.LCOWSupported() {
+		if system.LCOWSupported() {
```

Comment:
As mentioned on slack, it's a little moot as I'm actively working to remove all these markers anyway. But done. Push imminent.